x-pack/plugins/cloud_security_posture/public/components/fleet_extensions/inline_radio_group.tsx

```diff
@@ -0,0 +1,80 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React from 'react';
+import { useEuiTheme, EuiRadioGroup, type EuiRadioGroupProps } from '@elastic/eui';
+import { css } from '@emotion/react';
+
+type RadioGroupProps = Pick<EuiRadioGroupProps, 'onChange' | 'options' | 'idSelected'>;
+
+type Props = RadioGroupProps & {
+  size?: 's' | 'm';
+};
+
+export const InlineRadioGroup = ({ idSelected, size, options, onChange }: Props) => {
+  const { euiTheme } = useEuiTheme();
+
+  return (
+    <EuiRadioGroup
+      idSelected={idSelected}
+      options={options.map((o) => ({
+        id: o.id,
+        label: o.label,
+        disabled: o.disabled,
+        ['data-enabled']: idSelected === o.id,
+        ['data-disabled']: o.disabled,
+        className: '__extendedRadio',
+      }))}
```

Comment:
nit picks: 
1. consider creating this outside the JSX
2. maybe the classname should include the word `option` in it considering where it applies? `extendedRadioOption`?